src/BenchmarkDotNet/Diagnosers/NativeMemoryDescriptor.cs

```diff
@@ -0,0 +1,33 @@
+﻿// It is my Custom file header
+// BenchmarkDotNet
+// BenchmarkDotNetBenchmarkDotNet
+// NativeMemoryDescriptor.cs
+// Copyright  - 2019
+
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Reports;
+
+namespace BenchmarkDotNet.Diagnosers
+{
+    internal class NativeMemoryDescriptor : IMetricDescriptor
```

Comment:
nit: it would be nice to rename it to `AllocatedNativeMemoryDescriptor` to be consistent with the existing `AllocatedMemoryMetricDescriptor`